### ISSUE TITLE
Annotations only visible on moderator screen

### ISSUE THREAD

**Issue by c-store-cp** [bigbluebutton/bigbluebutton#11435]
**Describe the bug**
When using any tool to add annotations to a presentation, these annotations are only visible for presentator. Other participants see the cursor move, but only after logging out and back in, the annotations are visible.

**To Reproduce**
Open the default presentation, draw a line and watch from another login.

**Expected behavior**
All annotations should be visible for all participants.

**BBB version:**
BBB 2.2.31

**Desktop:**
Firefox 85 or Chrome 88 on Windows 10

**Comment by ffdixon** [comment]
We're interested to figure this out -- can you provide steps to reproduce?  Can you reproduce this on https://demo.bigbluebutton.org/?

**Comment by c-store-cp** [comment]
Hey Fred,

thanks for reacting so quickly. Yes, issue is reproduceable on the demo server. I have taken a screenshot and attached it.

![bbb](https://user-images.githubusercontent.com/19623013/108498915-d138c780-72ad-11eb-9185-63128dc1af09.jpg)

**Comment by hiroshisuga** [comment]
duplicate of #11430

**Comment by antobinary** [comment]
While this is indeed a duplicate of #11430 we should keep it open in case it provide any additional info

**Comment by c-store-cp** [comment]
After upgrading to 2.2.32, the issue seems to have gone away 👍

**Comment by c-store-cp** [comment]
Can Confirm on a second installation with 2.2.33 as fixed. Closing.
